src/Scanner.java

```diff
@@ -205,6 +209,41 @@
   }
 
   /**
+  * Returns boolean indicating if scanner is in reversed direction
+  */
+  public boolean getReversed(){
+    return is_reversed;
+  }
+
+  /**
+  * Specifies if the scan will be in reverse or not
+  * @param to_reverse Indication of scan direction. If this is not
+  * invoked, scanning will default to not being reversed.
+  *
+  * In a reversed scan, the first Scanner opened is opened on the region
+  * that the Scanner start key is in (the same as forward scan). But 
+  * subsequent scanners are opened via the overloaded 
+  * {@link #OpenScannerRequest} and the region to be opened on is 
+  * found through a META lookup using
+   * {@link HBaseClient#locateRegionClosestBeforeKey}.
+  * 
+  */
+  public void setReverse(){
```

Comment:
Great. Will address.
Actually javadoc is wrong even now, supposing there's some refactoring but javadoc was not fixed.